src/fsharp/FSharp.Core.Unittests/FSharp.Core/PrimTypes.fs

```diff
@@ -104,7 +104,7 @@ type LanguagePrimitivesModule() =
         
         // reference type
         let resultRef = LanguagePrimitives.GenericComparison "ABC" "ABCDE"
-        Assert.AreEqual(resultRef,-68)
+        Assert.AreEqual(sign resultRef,-1)
```

Comment:
Hmm, not a blocker, but the nunit APIs are authored to expect `Assert.AreEqual(expected, actual)` , whereas these are called the other way `Assert.AreEqual(actual, expected)`.  The failure message will be a little bit confusing.

Doesn't really matter, they were already wrong. In the future for new tests we can try to remember this though.